crates/vm/levm/README.md

```diff
@@ -18,6 +18,107 @@ Implementation of a simple Ethereum Virtual Machine in Rust.
 | Homestead      | ✅     |
 | Frontier       | ✅     |
 
+## Roadmap
+
+| Nº  | Milestone                       | Status |
+| --- | ------------------------------- | ------ |
+| 1   | Support Merge->Cancun forks     | ✅     |
+| 2   | Integrate `ethrex L1` <> `levm` | 🏗️     |
+| 3   | Support pre Merge forks         | 🏗️     |
+| 4   | Support Pectra upgrade          | 🏗️     |
+| 5   | Integrate `ethrex L2` <> `levm` | ❌     |
+| 6   | Performance                     | 🏗️     |
+
+### Milestone 1: Support Merge->Cancun forks
+
+This is having the minimum implementation so all the Ethereum Foundation tests from the fork Merge (Paris) to Cancun pass.
+
+The acceptance criteria for this milestone is that:
+
+- All the Cancun tests pass.
+- All the Shanghai tests pass.
+- All the Merge tests pass.
+
+### Milestone 2: Integrate `ethrex L1` <> `levm`
+
+Once we support all the forks from Merge to Cancun, we will integrate `ethrex L1` with `levm`.
+
+Nowadays `ethrex L1` uses `revm` as the backend VM. We will replace `revm` with `levm` and make sure all the tests pass.
+
+The acceptance criteria for this milestone is that:
+
+- All the Hive tests that pass with `revm` also pass with `levm`.
+- We have an insurance that if `levm` some time diverges from `revm`'s behavior, we will know it. And in such case, switching to `revm` while we fix the issue is easy.
```

Comment:
As an alternative, we could execute both revm and levm in parallel to ensure that the output is exactly equal. If this is not hard to implement it seems it would be the ideal solution until we fully adopt levm.

Reply:
WDYT?

Reply:
I agree. With @fkrause98 thought about a CI job that runs periodically and sends a message to the slack chanel in case of a divergence.